tests/parser/syntax/test_structs.py

```diff
@@ -421,22 +420,6 @@ def foo():
     ),
     (
         """
-struct X:
-    bar: int128
-    decimal: int128
```

Comment:
i guess these are allowed now?

Reply:
yes they are, similar to `def decimal(): pass`